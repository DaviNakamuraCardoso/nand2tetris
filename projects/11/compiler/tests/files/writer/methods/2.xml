<keyword>method</keyword>
<identifier>int</identifier>
<identifier>print</identifier>
<symbol>(</symbol>
<symbol>)</symbol>
<symbol>{</symbol>
<keyword>var</keyword>
<keyword>int</keyword>
<identifier>x</identifier>
<symbol>;</symbol>
<keyword>let</keyword>
<identifier>x</identifier>
<symbol>=</symbol>
<integerConstant>20</integerConstant>
<symbol>;</symbol>
<keyword>return</keyword>
<identifier>x</identifier>
<symbol>;</symbol>
<symbol>}</symbol>
