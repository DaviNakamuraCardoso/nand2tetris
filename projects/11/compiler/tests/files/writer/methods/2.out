function Point.print 1
push argument 0
pop pointer 0
push constant 20
pop local 0
push local 0
return
